### DISCUSSION TITLE
Add osu!mania scroll speed to replay / score display data

### DISCUSSION THREAD

**Discussion by GrilledCheeese** [ppy/osu#13178] [Ideas]
It would be an interesting statistic to quickly see at what speed everyone is playing for particular maps and difficulties, also making it easier to spot oddballs and possibly bugged replays that somehow show 40 scrollspeed or lazy cheaters that don't bother changing their scrollspeed from 12.
It could be put in between the users name and their max combo or wherever you guys feel it fits best
You can abbreviate it to S. Speed or just Speed
Example:
![image](https://user-images.githubusercontent.com/34809632/105682790-2a058200-5ef3-11eb-91c0-bbb4d00eed91.png)

**Comment by abraker95** [comment]
Scroll speed doesn't tell much. If your stage is thin you would play on lower scroll speed. If your stage is wide you would play on higher scroll speed. My stage covers 2/3 of the screen width and I play on 33 scrollspeed. Players on the default skin play on around 12 to 14 iirc.

**Comment by peppy** [comment]
There are many issues with this.

- It's not stored anywhere
- It's not guaranteed that the user isn't changing it
- It's a new feature which is best tracked on lazer, not osu-web (this won't be available on stable even if it's accepted as a feature we want)

I'll move to the lazer repo as a proposal for now.
